src/components/Registration.js

```diff
@@ -1,32 +1,36 @@
-import React, {useContext} from "react";
+import React, { useContext } from "react";
 import "../scss/Registration.scss";
-import {QuizContext} from "../QuizContext";
+import { QuizContext } from "../QuizContext";
 
-const Registration = () => {
+const Registration = ({quizStatus}) => {
 
-    const {setRegisteredUser} = useContext(QuizContext);
+  const {setRegisteredUser} = useContext(QuizContext);
 
-    const handleSubmit = (e) => {
-        e.preventDefault();
+  const handleSubmit = (e) => {
+    e.preventDefault();
 
-        setRegisteredUser(e.target.name.value);
-    };
+    setRegisteredUser(e.target.name.value);
+  };
 
   return (
-    <form className="form" onSubmit={handleSubmit}>
-
+    <form className="form"
+          onSubmit={handleSubmit}>
+      <h2>Zaraz zaczynamy</h2>{quizStatus !== "scheduled" &&
```

Comment:
Tu się raczej powinno sprawdzać, czy user nie jest zarejestrowany (z Contextu) bo quiz ma status scheduled i przed i po rejestracji, dopóki go się nie odpali z admina.
A poza tym wszystko spoko :)

Reply:
btw, żeby móc to normalnie potestować trzeba ustawić inny status dla mockowego quizu na scheduled, bo wtedy jeszcze się ten z backendu nie ładuje -> trzeba będzie to zmienić
QuizContext:
`   const [quizStatus, setQuizStatus] = useState("scheduled");`